.github/workflows/build.yml

```diff
@@ -9,12 +9,12 @@ jobs:
       COVERALLS_REPO_TOKEN: "cAEPfB2XY4L96cxO0SFaZeKdxwRCSNjZV"
     strategy:
       matrix:
-        os: [ubuntu-18.04, macos-10.15, windows-2019]
+        os: [ubuntu-1latest, macos-latest, windows-latest]
```

Comment:
should be `ubuntu-latest`

Reply:
Sorry for being the weirdo to review your 3 month old PR. It was late and I was being lazy 🙈 

I'll make a PR for this 👍